saleor/plugins/webhook/plugin.py

```diff
@@ -14,26 +14,24 @@
 from ...product.models import Collection, Product, ProductType, ProductVariant
 from ...webhook.event_types import WebhookEventAsyncType, WebhookEventSyncType
 from ...webhook.payloads import (
-    generate_checkout_payload,
     generate_collection_payload,
     generate_customer_payload,
     generate_excluded_shipping_methods_for_checkout_payload,
     generate_excluded_shipping_methods_for_order_payload,
     generate_fulfillment_payload,
     generate_invoice_payload,
     generate_list_gateways_payload,
-    generate_meta,
-    generate_order_payload,
     generate_page_payload,
     generate_payment_payload,
     generate_product_deleted_payload,
     generate_product_payload,
     generate_product_variant_payload,
     generate_product_variant_with_stock_payload,
-    generate_requestor,
     generate_sale_payload,
     generate_translation_payload,
 )
+from ...webhook.payloads_utils import generate_meta, generate_requestor
+from ...webhook.taxed_payloads import generate_checkout_payload, generate_order_payload
```

Comment:
I wouldn't call the module `taxed_payloads` if there's an option do `taxed=False`